packages/loader/container-loader/src/container.ts

```diff
@@ -174,43 +197,22 @@ export async function waitContainerToCatchUp(container: IContainer) {
     }
 
     return new Promise<boolean>((resolve, reject) => {
-        const deltaManager = container.deltaManager;
-
         container.on("closed", reject);
```

Comment:
Looks like we never clear this callback on successful path - might be worth fixing it.

Reply:
Suggested @Abe27342 take care of this in PR https://github.com/microsoft/FluidFramework/pull/10071 since he's changing the callback.

Reply:
And he did - yay.